dev-java/openjdk/openjdk-17.0.1_p12.ebuild

```diff
@@ -10,12 +10,18 @@ SLOT="$(ver_cut 1)"
 
 DESCRIPTION="Open source implementation of the Java programming language"
 HOMEPAGE="https://openjdk.java.net"
-SRC_URI="https://github.com/openjdk/jdk${SLOT}u/archive/refs/tags/jdk-${MY_PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="
+	https://github.com/openjdk/jdk${SLOT}u/archive/refs/tags/jdk-${MY_PV}.tar.gz
+		-> ${P}.tar.gz
+	!system-bootstrap? (
+		ppc64? ( big-endian? ( https://dev.gentoo.org/~arthurzam/distfiles/dev-java/${PN}/${PN}-bootstrap-17.0.1_p12-ppc64.tar.xz ) )
```

Comment:
let's make version a variable somewhere after inherit line.
maybe even 2 variables.
like
```bash
PPC64BE_BOOTSTRAP_TAR="17.0.1_p12"
X86_BOOTSTRAP_TAR="17.0.1_p12"
```
and reuse that in `SRC_URI` and in `pkg_setup`